docs/dev/setup/jenkins-gitlab.rst

```diff
@@ -385,8 +385,47 @@ Jenkins Server Setup
    ::
 
        echo "client_max_body_size 16m;" > client_max_body_size.conf
+5. The NGINX default timeout is pretty low. For plagarism check and unlocking student repos for the exam a higher timeout is advisable. Therefore we write our own nginx.conf and load it in the container.
+   ::
+
+       echo "
```

Comment:
Does anybody know how I would escape the content here or any idea how to better set the timeout in nginx? :thinking: